package.json

```
@@ -35,6 +35,7 @@
   "dependencies": {
     "dayjs": "1.11.7",
     "flatpickr": "4.6.13",
+    "he": "^1.2.0",
```

Версии должны фиксироваться без модификаторов
```suggestion
    "he": "1.2.0",
```